Habicus/Habicus-Core
# HABI-11 Adding Persistence Store for Dev DB

![](https://camo.githubusercontent.com/cf373500dfe17773807191f7fb730ea7ea4b959d/68747470733a2f2f692e696d6775722e636f6d2f5878585a6b6d4f2e706e67)

resolves #8 

## Review Process:

##### Core Reviewers:
@Habicus/core-team

##### Merge Owner:
@mweser
@Habicus/core-team
@austinsorrells
@schachte

## Status
**READY*

## Description
Adds a persistence store for the embedded test DB using JaxB annotations + XML from test resources dir. 


## Completed The Following (If Relevant)

## Steps to Test or Reproduce
1. Start server : `gradle -Dspring.profiles.active=development bootRun`
2. Go to `localhost:7777/h2`
3. Login with admin/admin
4. Run a global query on the USER database to ensure results are apparent
